packages/backend/src/modules/verifiers/repositories/VerifierStatusRepository.ts

```diff
@@ -0,0 +1,77 @@
+import { Logger } from '@l2beat/backend-tools'
+import { ChainId, UnixTime } from '@l2beat/shared-pure'
+import { VerifierStatusRow } from 'knex/types/tables'
```

Comment:
IMO it is easier to read if this type is also in the Repository .ts file

Reply:
On the other hand Row types are heavily coupled with knex table definitions so I would prefer to keep it here.  What's more I would like to prevent referencing modules here as it creates circular dependency: modules => peripherals => modules